dietpi/dietpi-software

```diff
@@ -9878,6 +9921,46 @@ _EOF_
 
 		fi
 
+		software_id=186 # IPFS Node
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			[[ -d '/mnt/dietpi_userdata/ipfs' ]] || G_EXEC mkdir /mnt/dietpi_userdata/ipfs
+			Create_User -d /mnt/dietpi_userdata/ipfs -G dietpi ipfs
+
+			cat << '_EOF_' > /etc/systemd/system/ipfs.service
+[Unit]
+Description=IPFS (DietPi)
+Wants=network-online.target
+After=network-online.target dietpi-boot.service
+
+[Service]
+User=ipfs
+Environment=IPFS_PATH=/mnt/dietpi_userdata/ipfs
+ExecStart=/usr/local/bin/ipfs daemon
+
+[Install]
+WantedBy=multi-user.target
+_EOF_
+			
+			if [[ ! -f '/mnt/dietpi_userdata/ipfs/config' ]]; then
+
+				IPFS_PATH='/mnt/dietpi_userdata/ipfs' G_EXEC_OUTPUT=1 G_EXEC ipfs init
+
+				IPFS_PATH='/mnt/dietpi_userdata/ipfs' G_EXEC ipfs config --json Addresses.Gateway '"/ip4/0.0.0.0/tcp/8087"'
+				IPFS_PATH='/mnt/dietpi_userdata/ipfs' G_EXEC ipfs config --json Addresses.API '"/ip4/0.0.0.0/tcp/5003"'
+				IPFS_PATH='/mnt/dietpi_userdata/ipfs' G_EXEC ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin '["*"]'
+
+				G_EXEC chown -R ipfs:dietpi /mnt/dietpi_userdata/ipfs
+				G_EXEC chmod -R 774 /mnt/dietpi_userdata/ipfs
+
+			fi
+
+			G_EXEC eval "echo -e '#!/bin/dash\nexport IPFS_PATH=/mnt/dietpi_userdata/ipfs' > /etc/bashrc.d/ipfs.sh"
```

Comment:
Please rename it to `/etc/bashrc.d/dietpi-ipfs.sh`. I aim to have all files that we create, where possible, prefixed with `dietpi-` so that they can be easily identified 🙂.